### DISCUSSION TITLE
pseudo-classes

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#60] [Q&A] [answered]
What is the use of pseudo-classes in CSS?

**Comment by Bunny77K9** [comment, marked as answer]
Pseudo-classes are used as a popular technique in CSS to change the style of an element when this element changes its state.

There are numerous examples of when you use a pseudo-class:

    For the style change when the mouse moves over the element
    For out-of-focus animations
    For providing styles for external links

We have to understand media queries in the next set of interview questions for web developers. Let’s check it out.
